ApiDoctor.Validation/ApiDoctor.Validation.csproj

```diff
@@ -32,9 +32,18 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Data.Edm, Version=5.8.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
do we need to add an odata dependency? we've avoided it in the past

Reply:
Fixed